packages/app/src/systems/Core/components/Providers.tsx

```diff
@@ -16,7 +16,8 @@ type AppProps = {
   children?: ReactNode;
 };
 
-const IS_TEST = process.env.NODE_ENV === "test";
+// const IS_TEST = process.env.NODE_ENV === "test";
+const IS_TEST = false;
```

Comment:
Should re-enable this?

Reply:
The issue I was having with this is whenever we run from a test env then `justContent=true` this causes the header to not be loaded and causes the e2e tests to fail bc they cannot click on the header buttons to navigate.